apps/omg_watcher_rpc/lib/eventer.ex

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherRPC.Eventer do
```

Comment:
Two options:

1/ pick a different name for this - something related to Phoenix.Channels which this here specifically uses + remove the copy-pasted module doc (write one which describes the intention behind this new module)
2/ move the original `Eventer` to `OMG.WatcherRPC`, which no modifications. The Eventer intends to translate internal events to Phoenix.Channel events, so I think such architecture is warranted.

I think (2/) is better, simpler at least

Reply:
2 is actually not possible to do - except if we have two same Core modules - because omg_watcher takes omg watcher rpc as dependency.

Reply:
ah bummer, it's this way around.

then I guess it's (1/) and lets keep this here as lightweight as possible.

Reply:
the module doc still - `OMG.WatcherRPC.BroadcastEvent` is now a thin layer forwarding the processed events into the Phoenix Channel pipe.